acinclude/win32-sspi.m4

```diff
@@ -0,0 +1,38 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+# check if the Windows SSPI requirements are available and work.
+# first argument is the variable containing the result
+#   (will be set to "yes" or "no")
+# second argument is the commands to run on success
+#
```

Comment:
The description got stale:

```suggestion
# Checks whether the Windows SSPI requirements are available and work.
# Sets squid_cv_win32_sspi to "yes" or "no".
# The argument is the commands to run on success.
```

The above suggestion also removes an empty trailing line in that comment. I do not insist on that removal because we are very inconsistent with regard to these comments styling (including `#`, `##`, and `dnl` variants and these empty lines), but I think it is better to converge towards no empty lines because our C++ descriptions do not use them and because they introduce an arguably unwanted distance from the thing being described, increasing the chances of description/code separation.

I also recommend rephrasing "requirements are available and work", but I do not know enough about SSPI to suggest specific improvements -- the macro code does not make the "work" part obvious (to me).